docs/design/StratisSoftwareDesign.lyx

```diff
@@ -3357,7 +3357,7 @@ size
 \begin_inset Text
 
 \begin_layout Plain Layout
-number
+integer
```

Comment:
Yeah, I was using this recommended tutorial: https://spacetelescope.github.io/understanding-json-schema/, which believes in integer type.